### ISSUE TITLE
set workspaceFolder other than /com.docker.devenvironments.code

### ISSUE THREAD

**Issue by jfedotov** [docker/dev-environments#98]
I would like to connect to a remote dev container and have access to files stored inside a container. However, every time I open a dev container via Docker's Dev Environment the "workspaceFolder" is always set to "/com.docker.devenvironments.code" which is mounted to a local folder on my PC.
Is any way how to change workspaceFolder to the path inside the container? Because in my case all source files are git cloned into the container. I do not store source files locally on PC, everything is inside the dev container.
Currently, I can manually change workspaceFolder at "Open Container Configuration file" in VScode. 
I tried to add workspaceFolder into .devcontainer.json, but seems Docker's Dev Environment ignores devcontainer.json.

**Comment by jfedotov** [comment]
<img width="1021" alt="Screenshot 2022-01-28 at 21 26 16" src="https://user-images.githubusercontent.com/19622412/151616926-aa1cdf06-61a2-46a9-8d97-c1bf3456007f.png">

**Comment by glours** [comment]
Hello @jfedotov 

ATM there is no way to configure the source code folder, we're looking to use Compose format to configure you Dev Environment which may allow you to change the source folder name. 
We don't support the `.devcontainer.json` configuration now but it's not exclude in the future

**Comment by kb-ig** [comment]
Yeah I need this option. I have my VScode title formatted so that the name of the folder is the first part of the title, so that when I have multiple windows open (which is pretty much all day every day), I can easily identify which is which. At the moment, when running multiple dev containers, I have no easy way of identifying this. 

![image](https://user-images.githubusercontent.com/91877080/154230265-4ec843e9-25e6-43a4-948e-31f33511a598.png) 🤔 

I'm sure there's a good reason for the workspace folder to be `/com.docker.devenvironments.code`, but I think it would be better if the default was for every container either used a more unique workspace folder (perhaps the container name or even just the repo name).

**Comment by rumpl** [comment]
> I'm sure there's a good reason for the workspace folder to be `/com.docker.devenvironments.code`

The main reason is that it's a unique (and yes, weird) name so that there is very little chance it would already exist in the image. You could have a repo name called `etc`, and the container name changes so it's not a great fit.

**Comment by kb-ig** [comment]
> > I'm sure there's a good reason for the workspace folder to be `/com.docker.devenvironments.code`
> 
> The main reason is that it's a unique (and yes, weird) name so that there is very little chance it would already exist in the image. You could have a repo name called `etc`, and the container name changes so it's not a great fit.

Yeah that's a fair point. Maybe prefixing the unique folder name with the name of the repo would solve this issue. Best of both worlds.

**Comment by benknight135** [comment]
I was able to get around this for the time being by creating a symlink where I wanted the folder so the ```/com.docker.devenvironments.code``` is still created as normal but is also in the location required and all changes in the devenvironments folder are mirrored in the target folder.
In my dockerfile it looks like this:
```dockerfile
RUN ln -s /com.docker.devenvironments.code /path/to/folder
```
